setup.py

```diff
@@ -2,14 +2,22 @@
 SPDX-License-Identifier: Apache-2.0
 Copyright 2017 Massachusetts Institute of Technology.
 '''
+import os
+import shutil
+
 import setuptools
 
+
 with open('README.md', encoding="utf-8") as fh:
     long_description = fh.read()
 
+# Copy keylime.conf temporarily into the keylime folder.
+# This is done in oder to include it as a package resource because data_files is deprecated.
+shutil.copy("keylime.conf", "keylime")
 
 setuptools.setup(
     name='keylime',
+    version='6.4.0',
```

Comment:
Let's not bump this up just yet. I expect we'll have a 6.3.1 shortly since we have a lot of minor improvements recently.

Reply:
+1